static/css/restyle/restyle.less

```diff
@@ -2074,4 +2074,51 @@ body.editor-tools {
 .listing .item .desc {
   color: #777;
   font-size: 1rem;
+
+/* Source/Diff viewer */
+body.file-viewer {
+  pre, code, kbd, tt, samp, tt {
+    font-family: @mono-stack !important;
+  }
+
+  #files {
+    width: 18em;
+
+    #files-tree {
+      a > span {
+        border-radius: 0;
+      }
+
+      .notice {
+        border-top: none;
+      }
+
+      /* Notice. */
+      #file-viewer.messages-duplicate #files a.notice-duplicate > span:after,
+      #file-viewer.messages-signing #files a.notice-signing > span:after,
+      #file-viewer.messages-all #files a.notice > span:after,
+      /* Warning. */
+      #file-viewer.messages-duplicate #files a.warning-duplicate > span:after,
+      #file-viewer.messages-signing #files a.warning-signing > span:after,
+      #file-viewer.messages-all #files a.warning > span:after,
+      /* Error. */
+      #file-viewer.messages-duplicate #files a.error-duplicate > span:after,
+      #files a.error-signing > span:after,
+      #files a.error > span:after {
+          vertical-align: baseline;
+      }
+    }
+  }
+
+  #content-wrapper {
+    .notification-box {
+      border-radius: 0;
+      box-shadow: none;
+    }
+  }
+
+  #content-wrapper, #messages, #thinking, #validation-options {
```

Comment:
Please can we have a newline after each `,` so we have a selector on one line - it makes it easier to read.